src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/LicenseMetadata.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Packaging.Licenses;
+using NuGet.Shared;
+
+namespace NuGet.Packaging
+{
+    public class LicenseMetadata : IEquatable<LicenseMetadata>
+    {
+        public static Version EmptyVersion = new Version(1, 0, 0);
+        public static Version CurrentVersion = new Version(1, 0, 0);
+        public static Uri DeprecateUrl = new Uri("https://aka.ms/deprecateLicenseUrl");
+
+        public LicenseType Type { get; }
+        public string License { get; }
+        public NuGetLicenseExpression LicenseExpression { get; }
+        public Version Version { get; }
+
+        public LicenseMetadata(LicenseType type, string license, NuGetLicenseExpression expression, Version version)
+        {
+            Type = type;
+            License = license ?? throw new ArgumentNullException(nameof(license));
```

Comment:
what if this was empty?

Reply:
Should be handled earlier in the code...this is consumption driven, I will investigate what we do on consumption if the license url is  null and mimic the behavior.

Reply:
This model is used both during pack and reading. 
When reading we don't know what tool people used to pack. 

So while empty is a problem, the license content does not really affect restore and it's only prominent whne you are installing/browsing a package. 
In those situation we will tell the customer that the package author messed and how (from empty expression to unparseable expressions).